synthesizer/process/src/verify_deployment.rs

```diff
@@ -23,10 +23,6 @@ impl<N: Network> Process<N> {
         rng: &mut R,
     ) -> Result<()> {
         let timer = timer!("Process::verify_deployment");
-        // Retrieve the program ID.
```

Comment:
This cannot be removed from here.

Local processes never use the VM to check, so this would skip a critical check.

Reply:
Reverted in [7660096](https://github.com/AleoHQ/snarkVM/pull/2270/commits/7660096acfbf4d47aa175a3ff502382b830f6f10)

Reply:
@ljedrz